DependencyInjection/Configuration.php

```diff
@@ -40,10 +40,24 @@ public function getConfigTreeBuilder()
                     ->requiresAtLeastOneElement()
                     ->useAttributeAsKey('name')
                     ->prototype('array')
-                        ->children()
+						->children()
                             ->scalarNode('title')->isRequired()->end()
                             ->scalarNode('description')->isRequired()->end()
-                            ->scalarNode('link')->isRequired()->end()
+							->arrayNode('link')
```

Comment:
Can you please update all the tabs with 4 spaces in order to be compliant with the PSR-2 code style please?

```
Code MUST use 4 spaces for indenting, not tabs.
```

See: http://www.php-fig.org/psr/psr-2/